compiler/modulepaths.nim

```diff
@@ -87,6 +87,10 @@ when false:
           result = findInNimbleDir(pkg, subdir, nimbleDir / "pkgs")
           if result.len > 0: return result
           when not defined(windows):
+            result = findInNimbleDir(pkg, subdir, "/usr/share/nimble/pkgs")
+            if result.len > 0: return result
+            result = findInNimbleDir(pkg, subdir, "/usr/local/share/nimble/pkgs")
```

Comment:
But what about "usr/share/local"?

Reply:
We currently use /usr/share or /usr/local/share in install and deinstall scripts, so I added probes for knows as known locations. Do you want me to add /usr/share/local as well in those scripts and here?

Reply:
Sorry I have no idea if "usr/share/local" exists XD, it was a bad joke.